cluster_migration_core/cluster_migration_core/clients/rest_client_default.py

```diff
@@ -53,6 +63,24 @@ def post_doc_to_index(self, port: int, index: str, doc: dict) -> ops.RESTRespons
 
         return ops.perform_post(rest_path=rest_path, data=json.dumps(doc), headers=headers, params=params)
 
+    def count_doc_in_index(self, port: int, index: str) -> ops.RESTResponse:
```

Comment:
Very minor nitpick: mind changing this to `count_docs_in_index()` (e.g. pluralizing doc)?

Reply:
Will do!